README.md

```diff
@@ -356,26 +353,11 @@ var response = await airbrake.NotifyAsync(ex);
 Console.WriteLine(response.Url);
 ```
 
-##### .NET 3.5
-
-In .NET 3.5 you can subscribe to the `NotifyCompleted` event and define your
-custom logic in an event handler:
-
-```csharp
-airbrake.NotifyCompleted += (sender, eventArgs) =>
-{
-    airbrakeResponse = eventArgs.Result;
-    Console.WriteLine(airbrakeResponse.Url);
-};
-
-airbrake.NotifyAsync(ex);
-```
-
 #### `AddFilter`
 
 A notice can be customized or ignored before it is sent to Airbrake via
 `AddFilter`. A lambda expression that is passed to the `AddFilter` method
-accepts a `Notice` that can be processed by your code. The `Notice` object is
```

Comment:
The diff says you removed an "a" from "accepts a `Notice`". I think it is OK to have it there.